pipeline/scripts/13-excelExport.R

```diff
@@ -137,9 +144,44 @@ if (z_score == 1) {
   }
   colnames(outlist)[startcol:ncol(outlist)] <- cnames.z
   
+  # Extra output: calculate robust scaler (Zscores minus outliers in Controls)
+  # calculate mean and sd for Control group without outliers
+  outlist.noZ$avg.ctrls <- 0
+  outlist.noZ$sd.ctrls  <- 0
+  
+  robustscaler <- function(x, perc=5, control_col_ids) {
```

Comment:
Het is normaal gesproken de bedoeling dat parameters met een default waarde al laatste komen, dus:
`function(x, control_col_ids, perc=5)`

Je kan dan de functie op deze manier aanroepen:
robustscaler(x, control_col_ids)

Reply:
Aangepast.